nixos/roles/ceph/rgw.nix

```diff
@@ -21,18 +21,33 @@ let
 
   cephPkgs = fclib.ceph.mkPkgs role.cephRelease;
 
+  radosListenPort = toString 7480;
+  haproxyPort = toString 7475;
+  vhostName = lib.concatStringsSep "." (
```

Comment:
I'd rather have this being a proper option where the computed value is the default and we can completely override this. Otherwise this appears to be a potential "let trap".